### ISSUE TITLE
Not Showing Correct Project for File being Edited

### ISSUE THREAD

**Issue by Bioblaze** [HelloWorld017/atom-discord#39]
Its not showing the actual project for the file currently being edited, only showing the first project int he tree-view. I fixed that with my git pull. #38

**Comment by HelloWorld017** [comment]
Closed by #38

**Comment by Ekidoxx** [comment]
Dunno whether I should write here or in the [pull request](https://github.com/HelloWorld017/atom-discord/pull/38), but once I have updated the package locally with content of this repository master-branch instead of checking it on package version `1.4.0`, I found a bug.

Generally, the whole path-showing works great as I already wrote in [other comment](https://github.com/HelloWorld017/atom-discord/pull/38#issuecomment-359630438), but only until the editor starts with showing up any tab that isn't a file, like, for example, settings.
Because then, the package fails to activate:
![image](https://user-images.githubusercontent.com/14035306/35257088-9ef96652-fff7-11e7-91ec-72b720f3ba07.png)

There's the content of Stack Trace:
```
TypeError: Cannot read property 'buffer' of undefined
    at createLoop (C:\Users\Ekidoxx\.atom\packages\atom-discord\src\index.js:90:50)
    at Object.activate (C:\Users\Ekidoxx\.atom\packages\atom-discord\src\index.js:112:3)
    at Package.module.exports.Package.activateNow (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package.js:255:25)
    at C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package.js:227:38
    at Package.module.exports.Package.measure (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package.js:99:21)
    at C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package.js:220:32
    at Package.module.exports.Package.activate (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package.js:217:40)
    at PackageManager.activatePackage (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package-manager.js:695:42)
    at config.transactAsync (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package-manager.js:670:36)
    at Config.module.exports.Config.transactAsync (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\config.js:367:24)
    at PackageManager.activatePackages (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package-manager.js:668:23)
    at PackageManager.activate (C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\package-manager.js:647:50)
    at C:\Users\Ekidoxx\AppData\Local\atom\app-1.23.1\resources\app\src\atom-environment.js:865:36
```
